ldap/servers/slapd/back-ldbm/db-mdb/mdb_config.c

```diff
@@ -531,6 +564,22 @@ dbmdb_ctx_t_setup_default(struct ldbminfo *li)
     }
 }
 
+/* Add a dummy config entry to ensure compatibility with
+ * applications that tries to tweak bdb parameters
+ */
+static void
+dbmdb_create_bdb_config_entry(struct ldbminfo *li)
+{
+    int hasdbhome = 0;
+    char *dbhome = dbmdb_get_home_dir(li, &hasdbhome);
+    char *entries[] = { (char*)bdb_config_entry_template, ""};
+
+    if (!hasdbhome) {
+        dbhome = "???";
+    }
+    dbmdb_ctx_t_add_dse_entries(li, entries, dbhome, dbhome, NULL, 0);
```

Comment:
Just a dummy question. Do mdb and bdb share the same dbhome ?

Reply:
Yes

Reply:
As the file names are differents there is no conflict